src/HasTags.php

```diff
@@ -119,9 +127,15 @@ public function scopeWithAllTagsOfAnyType(Builder $query, $tags): Builder
         return $query;
     }
 
-    public function scopeWithAnyTagsOfAnyType(Builder $query, $tags): Builder
+    /**
+     * @param Builder $query
```

Comment:
Please type everything you can and use fully qualified class name in this docblock (like the existing docblocks)

Reply:
Fixed, thank you